src/osd/scrubber/scrub_backend.cc

```diff
@@ -885,6 +867,7 @@ std::optional<std::string> ScrubBackend::compare_obj_in_maps(const hobject_t& ho
   auto [auths, objerrs] =
     match_in_shards(ho, auth_res, object_error, errstream);
 
+
   auto opt_ers =
     for_empty_auth_list(std::forward<std::list<pg_shard_t>>(auths),
```

Comment:
Why does this use std::forward?

I'm commenting on this, btw, because the first commit in this PR changes the whitespace here and the last commit changes it back.  In the future, please squash changes like that back to the commit in which they originated so that there aren't phantom changes in the branch.

Reply:
the std::fwd is a relic from a previous version (where I used a tuple). Will change.